go/common/crypto/drbg/hmac_drbg_test.go

```diff
@@ -36,7 +36,7 @@ func TestDrbgNISTExample1(t *testing.T) {
 	)
 
 	// HMAC_DRBG_Generate
-	v, err := drbg.generate(readSz, nil)
+	v, err := drbg.generate(readSz, []byte{})
```

Comment:
Should we test both `nil` and `[]byte{}` in tests? They should behave identically now.

Reply:
Yeah I would test both just to prevent regressions.